<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logo_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Safatique logo (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-20</y>
      <width>1041</width>
      <height>811</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>backgroundDBMS.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_CustomerInfo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>681</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Customer Information</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_newExpiry">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_username_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Expiry</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_newCard">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_savedCard">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Saved Card Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lastName">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_address">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_newAddress">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_firstName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_COD">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cash on Delivery</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CustomerEmail">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_username_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>New Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CustomerAddress">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_savedAddress">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Saved Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CustomerContact">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_username_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Card Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CustomerLastName">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_username_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CVC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_newCVC">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_CustomerFirstName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_orderDetails">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>841</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Order Details</string>
    </property>
    <widget class="QTableWidget" name="ProductsTableWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>761</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ProductID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit Price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Price</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="label_TotalAmount">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>670</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TotalAmt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>670</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Payable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_placeOrder">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Place Order</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>label_logo_7</zorder>
   <zorder>groupBox_CustomerInfo</zorder>
   <zorder>groupBox_orderDetails</zorder>
   <zorder>label_TotalAmount</zorder>
   <zorder>label_username_10</zorder>
   <zorder>pushButton_placeOrder</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
